contracts/modules/andromeda-rates/src/contract.rs

```diff
@@ -88,6 +89,12 @@ fn execute_set_rate(
     );
     // Validate the local rate's value
     rate.value.validate()?;
+
+    // Set the sender as the recipient in case no recipients were provided
+    if rate.recipients.is_empty() {
```

Comment:
We should also consider rate in `instantiate` function as well.

Reply:
The `validate` method checks for valid `LocalRateValue`